cmake/modules/Findrubberband.cmake

```diff
@@ -85,14 +85,32 @@ if(rubberband_FOUND)
         INTERFACE_COMPILE_OPTIONS "${PC_rubberband_CFLAGS_OTHER}"
         INTERFACE_INCLUDE_DIRECTORIES "${rubberband_INCLUDE_DIR}"
     )
-    is_static_library(rubberband_IS_STATIC Chromaprint::Chromaprint)
+    is_static_library(rubberband_IS_STATIC rubberband::rubberband)
     if(rubberband_IS_STATIC)
-      find_package(FFTW REQUIRED)
       find_library(SAMPLERATE_LIBRARY samplerate REQUIRED)
       set_property(TARGET rubberband::rubberband APPEND PROPERTY INTERFACE_LINK_LIBRARIES
-        FFTW::FFTW
         ${SAMPLERATE_LIBRARY}
       )
+      find_package(FFTW)
+      if (FFTW_FOUND)
+        set_property(TARGET rubberband::rubberband APPEND PROPERTY INTERFACE_LINK_LIBRARIES
+          FFTW::FFTW
+        )
+      endif()
+      find_package(FFTW)
+      if (FFTW_FOUND)
+        set_property(TARGET rubberband::rubberband APPEND PROPERTY INTERFACE_LINK_LIBRARIES
+          FFTW::FFTW
+        )
+      endif()
+      find_package(Sleef)
+      if (Sleef_FOUND)
+        find_library(sleefdft_path sleefdft REQUIRED)
+        set_property(TARGET rubberband::rubberband APPEND PROPERTY INTERFACE_LINK_LIBRARIES
+          Sleef::sleef
+          ${sleefdft_path}
+        )
+      endif()
```

Comment:
whats the error like if neither sleef nor fftw is found?

Reply:
Rubberband is linked one of it, which is the choice of the package provider. If the library is not found build will fail with a significant linker error. 
I have no idea how to investigate which library is used to fail early. 
We need to wait for the Rubberband CONFIG mode.

Reply:
Couldn't we just detect if `!fftw_FOUND && !Sleef_FOUND` and error out early then? Sorry for the dumb questions, I'm not familiar enough with cmake. I'm just trying to wrap my head around it.